app/src/components/buttons/toggleButton/toggleButton.scss

```diff
@@ -0,0 +1,48 @@
+.toggle-button {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+.button-item {
+  display: table;
+  table-layout: fixed;
+  float: left;
+  height: 100%;
+  padding: 0 5px;
+  box-sizing: border-box;
+  border: 1px solid $COLOR--gray-80;
+  background-color: $COLOR--white-two;
+  text-align: center;
+  font-family: $FONT-REGULAR;
+  font-size: 13px;
+  color: $COLOR--charcoal-grey;
+  cursor: pointer;
+
+  &.one-of-1 {
+    width: 100%;
+  }
+  &.one-of-2 {
+    width: 50%;
+  }
+  &.one-of-3 {
+    width: 33.33333%;
+  }
+  &.one-of-4 {
```

Comment:
What if we have 5 items? Maybe we could add width in 'style' sttribute in JSX depending on item count?